.circleci/config.yml

```diff
@@ -94,6 +120,33 @@ workflows:
     jobs:
       - build
 
+      - docker/publish:
+          name: Publish master docker image
+          executor: docker/machine
+          context: dockerhub
+          image: banzaicloud/istio-operator
+          tag: master
+          requires:
+            - build
+          filters:
+            branches:
+              only: master
```

Comment:
We don't use `master`, I think this is not needed.

Reply:
I think this job could be run for all release-* branches instead.
I'm not sure what would be a good docker tag, though. E.g. release-1.5 is not a good tag. Maybe dev-1.5?

Reply:
I don't think that's needed, imho there is no need to push new dev docker images on every single release branch push, it's just enough on tags.